### DISCUSSION TITLE
Donation

### DISCUSSION THREAD

**Discussion by geocine** [kohya-ss/sd-scripts#35] [Ideas]
Consider opening a Patreon or other means of receiving financial support for your open-source project. Maintaining an open-source project can be challenging, especially while also working a full-time job. Your willingness to engage with users and share your knowledge is valuable. Users who are passionate about the project and use it frequently may consider donating to show their appreciation and support the maintainer.

**Comment by kohya-ss** [comment]
Thank you for your suggestion!
I only update for my fun and I don't know when I will stop updating. However, I am very happy that someone is willing to support me.

note.com has a donation function, but it is in Japanese only and requires an account, so I think it might be difficult to use.

Patreon seems to be better, but I think the subscriptions do not seem very appropriate.
